.travis.yml

```diff
@@ -5,28 +5,33 @@ os:
   - linux
   - osx
 
+git:
+  depth: 10
+  submodules: false
+
 go:
   - 1.8
 
 env:
```

Comment:
I think it's worth preserving a non-`-race` variant.

Reply:
Done.